tracetools/src/tracetools.c

```diff
@@ -489,6 +484,64 @@ DEFINE_TRACEPOINT(
   TRACEPOINT_ARGS(
     buffer))
 
+#ifndef TRACETOOLS_TRACEPOINTS_EXCLUDED
+static void * tracetools_provider_handle = NULL;
+
+bool ros_trace_compile_status(void)
+{
+  return true;
+}
+
+bool ros_trace_runtime_status(void)
+{
+  return tracetools_provider_handle != NULL;
+}
+
+static bool get_boolean_env(const char *name)
+{
+  const char * value = getenv(name);
+  return value != NULL && strcmp(value, "1") == 0;
```

Comment:
Concerning this line here and in `mark_process.cpp`:

I'm kind of used to env vars enabling a functionality if they're simply set and non-empty, instead of specifically requiring something like `"1"`. What do you think?

Reply:
I don't have a strong opinion. Here, I just replicated the semantics of the old [`ROS_LOCALHOST_ONLY` envvar](https://github.com/ros2/rcl/blob/ca440022a4fec2ce4d3624e5412bd22bc73b247f/rcl/src/rcl/localhost.c#L44-L47). We can remove the equality requirement.

Reply:
Yeah okay, we seem to use `=1` and `=0` in many places (e.g., [for logging env vars](https://docs.ros.org/en/rolling/Concepts/Intermediate/About-Logging.html#environment-variables)), and we even have `rcutils_get_env_var_zero_or_one()`. Let's keep this, but then I would make it explicit in the README:

```diff
-To avoid loading the tracer at runtime (and therefore disable all instrumentation), set the `TRACETOOLS_RUNTIME_DISABLE` environment variable:
+To avoid loading the tracer at runtime (and therefore disable all instrumentation), set the `TRACETOOLS_RUNTIME_DISABLE` environment variable to `1`:
```

Reply:
Done in https://github.com/ros2/ros2_tracing/pull/185/commits/9bba68582b295e1e13a3fa3549bdba540abb9386